packages/file-upload/src/from-url/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react';
+import { Box, TextField, Typography } from '@mui/material';
+import { FromURLContainer } from './styles';
+import { Button } from '../../../button/src/index';
+
+export interface FromUrlProps {
+  accept?: string;
+  acceptLabels?: string;
+  variant?: 'single' | 'combined';
+  componentDisabled?: boolean;
+  buttonDisabled?: boolean;
+  inputDisabled?: boolean;
```

Comment:
maybe we can take an approach like mui here

disabled: disable all component
buttonProps: all props to a button including a disabled option
TextFieldProps: same props of TextField including disable

and we spread the props inside of each component as well